Makefile

```diff
@@ -27,7 +27,7 @@ export DOCKER_IMAGE_NAME ?= $(DOCKER_IMAGE):$(DOCKER_TAG)
 export DOCKER_BUILD_FLAGS =
 
 ifeq ($(wildcard /.dockerenv),)
-export DOCKER_RUN := docker run -it --rm -v $(CURDIR):/src -p $(HUGO_PORT):$(HUGO_PORT) -e YARN_BUILD_DISABLED -e GITHUB_BASIC_AUTH $(DOCKER_IMAGE_NAME)
```

Comment:
So, locally, we'll want to have it be interactive (so we can hit `^C`). 

Let' instead add `DOCKER_RUN_FLAGS`